src/auto/enums.rs

```diff
@@ -494,3 +494,113 @@ impl FromGlib<glib_sys::GTimeType> for TimeType {
         }
     }
 }
+
+#[cfg(any(feature = "v2_66", feature = "dox"))]
```

Comment:
Need to add `v2_66` feature also to `Cargo.toml`.

Reply:
This was also forgotten for other crates I guess. Can you create PRs for that? For gio I created https://github.com/gtk-rs/gio/pull/319 already.

Reply:
AH somehow I missed this one. I will update the relevant crates